### ISSUE TITLE
Cannot reset password

### ISSUE THREAD

**Issue by cblavier** [smpallen99/coherence#274]
Hey,

I'm using coherence 0.4.0 along with phoenix 1.2.4 and I have issues when sending reset instructions email.

I can go to the `/passwords/new` route, but when I submit the form with my email, it raises the following exception :

```
[error] #PID<0.544.0> running PhenixDisplays.Endpoint terminated
Server: localhost:5000 (http)
Request: POST /passwords
** (exit) an exception was raised:
    ** (UndefinedFunctionError) function Helpers.password_url/3 is undefined (module Helpers is not available)

...

 (coherence) lib/coherence/controllers/password_controller.ex:60: Coherence.PasswordController.create/2
        (coherence) lib/coherence/controllers/password_controller.ex:1: Coherence.PasswordController.action/2
        (coherence) lib/coherence/controllers/password_controller.ex:1: Coherence.PasswordController.phoenix_controller_pipeline/2
```

Thanks for your help

**Comment by cblavier** [comment]
I solved my issue. I had to add `router: MyApp.Router` to coherence config in config.exs.

Then I met another error that could be solved by merging #240 on 0.4.0. 
Can you create a 0.4.1?

**Comment by pdgonzalez872** [comment]
@cblavier This is no longer valid correct? If so, could you please close the issue? Thanks!
